Mathlib/CategoryTheory/Limits/Shapes/Diagonal.lean

```diff
@@ -129,61 +156,52 @@ def pullbackDiagonalMapIso :
           (by simp only [Category.assoc, condition])
           (by simp only [Category.assoc, condition])) ≅
       pullback i₁ i₂ where
-  hom :=
-    pullback.lift (pullback.snd _ _ ≫ pullback.fst _ _) (pullback.snd _ _ ≫ pullback.snd _ _) (by
-      ext
-      · simp [Category.assoc, pullback_diagonal_map_snd_fst_fst, pullback_diagonal_map_snd_snd_fst]
-      · simp [Category.assoc, pullback.condition, pullback.condition_assoc])
-  inv :=
-    pullback.lift (pullback.fst _ _ ≫ i₁ ≫ pullback.fst _ _)
-      (pullback.map _ _ _ _ (𝟙 _) (𝟙 _) (pullback.snd _ _) (Category.id_comp _).symm
-        (Category.id_comp _).symm) (by
-        ext
-        · simp only [Category.assoc, diagonal_fst, Category.comp_id, limit.lift_π,
-            PullbackCone.mk_pt, PullbackCone.mk_π_app, limit.lift_π_assoc, cospan_left]
-        · simp only [condition_assoc, Category.assoc, diagonal_snd, Category.comp_id,
-            limit.lift_π, PullbackCone.mk_pt, PullbackCone.mk_π_app,
-            limit.lift_π_assoc, cospan_right])
+  hom := pullbackDiagonalMapIso_hom f i i₁ i₂
+  inv := pullbackDiagonalMapIso_inv f i i₁ i₂
+  hom_inv_id := by
+    dsimp only [pullbackDiagonalMapIso_hom, pullbackDiagonalMapIso_inv]; aesop_cat
+  inv_hom_id := by
+    dsimp only [pullbackDiagonalMapIso_inv, pullbackDiagonalMapIso_hom]; aesop_cat
```

Comment:
Should this decl now be tagged with `@[simps]`? I.e. on L153

Reply:
I made pullbackDiagonalMapIso_hom an abbrev now, maybe that's good enough? The only reason I factored them out was for peformance reasons anyways